src/libvfcinstrument/libVFCInstrument.cpp

```diff
@@ -122,6 +124,49 @@ struct VfclibInst : public ModulePass {
 
   VfclibInst() : ModulePass(ID) {}
 
+  // Taken from
+  // https://www.fluentcpp.com/2017/04/21/how-to-split-a-string-in-c/
+  std::vector<std::string> split(const std::string &s, char delimiter) {
+    std::vector<std::string> tokens;
+    std::string token;
+    std::istringstream tokenStream(s);
+    while (std::getline(tokenStream, token, delimiter)) {
+      tokens.push_back(token);
+    }
+    return tokens;
+  }
+
+  StringRef getModuleName(Module &M) {
+    const std::string &sourceFilename = M.getModuleIdentifier();
+    /* Split the path with the separator */
+    std::vector<std::string> tokensDir = split(sourceFilename, '/');
```

Comment:
Can you `#define PATH_SEPARATOR "/" ? This way if we want to support windows later, it will be easy to port.

Reply:
Yes, I changed it. Btw, this constant should be added in future version of C++